BlueShift-iOS-SDK/BlueShiftAppData.m

```diff
@@ -46,6 +39,20 @@ - (NSString *)bundleIdentifier {
     return [[[NSBundle mainBundle] infoDictionary] objectForKey:(NSString *)kCFBundleIdentifierKey];
 }
 
+- (BOOL)enablePush {
+    NSString *val = [[NSUserDefaults standardUserDefaults] objectForKey:kBlueshiftEnablePush];
+    BOOL enablePush = YES;
+    if (val) {
+        enablePush = [val isEqual:@"YES"] ? YES : NO;
+    }
+    return enablePush;
+}
+
+- (void)setEnablePush:(BOOL)enablePush {
+    NSString *val =  enablePush ? @"YES" : @"NO";
```

Comment:
can we use 0 and 1 to make it simple when we compare them?

Reply:
In Objective C we use YES and NO for boolean values and "YES" "NO" is more readable while comparing it like enablePush = [val isEqual:@"YES"] ? YES : NO;